modelci/experimental/curd/model_train.py

```diff
@@ -42,7 +42,7 @@ def get_all() -> List[TrainingJob]:
 
 def save(training_job_in: TrainingJobIn) -> str:
     model_id = training_job_in.model
-    if not ModelDAO.exists_by_id(ObjectId(model_id)):
+    if not service.exists_by_id(str(model_id)):
```

Comment:
Is `exists_by_id` accept ObjectID as model id input? If no, you may add this ObjectId support